lib/AArch64Base.ml

```diff
@@ -2412,6 +2470,15 @@ let do_pp_instruction m =
       pp_mem "LDG" V64 rt rn (K k)
   | I_UDF k ->
       sprintf "UDF %s" (m.pp_k k)
+  (* Pointer Authentication Code *)
+  | I_PAC (key, r1, r2) ->
+      sprintf "PAC%s %s, %s" (PAC.pp_upper_key key) (pp_reg r1) (pp_reg r2)
+  | I_AUT (key, r1, r2) ->
+      sprintf "AUT%s %s, %s" (PAC.pp_upper_key key) (pp_reg r1) (pp_reg r2)
+  | I_XPACI r ->
+      sprintf "XPACI %s" (pp_reg r)
+  | I_XPACD r ->
+      sprintf "XPACD %s" (pp_reg r)
```

Comment:
Is there a way you had to split this into XPACD and XPACI rather than just XPAC much like PAC.